pkg/specgenutil/volumes.go

```diff
@@ -691,3 +691,9 @@ func validChownFlag(flag string) (bool, error) {
 
 	return true, nil
 }
+
+// On Unix systems this is a no-op, but on Windows, ensures the path retains unix seprators
+// (since filepath.Clean converts Unix segments to filepath.Separator)
+func unixPathClean(path string) string {
+	return filepath.ToSlash(filepath.Clean(path))
```

Comment:
CAn you just use the `path` package instead of `filepath`?

Reply:
yes should have just done that